packages/components/tests/integration/components/hds/pagination/index-test.js

```diff
@@ -1,17 +1,201 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { render } from '@ember/test-helpers';
+import { click, render, select } from '@ember/test-helpers';
 import { hbs } from 'ember-cli-htmlbars';
 
 module('Integration | Component | hds/pagination/index', function (hooks) {
   setupRenderingTest(hooks);
 
+  // Test rendering:
+
   test('it renders the component', async function (assert) {
-    await render(hbs`<Hds::Pagination />`);
+    await render(hbs`
+      <Hds::Pagination @totalItems={{100}} @itemsPerPage={{10}} @currentPage={{1}} as |P|>
+        <P.Nav />
+      </Hds::Pagination>
+    `);
     assert.dom(this.element).exists();
   });
+
   test('it should render with a CSS class that matches the component name', async function (assert) {
-    await render(hbs`<Hds::Pagination id="test-pagination" />`);
-    assert.dom('#test-pagination').hasClass('hds-pagination');
+    await render(hbs`
+      <Hds::Pagination @totalItems={{100}} @itemsPerPage={{10}} @currentPage={{1}} id="test-pagination-bar" as |P|>
+        <P.Nav />
+      </Hds::Pagination>
+    `);
+    assert.dom('#test-pagination-bar').hasClass('hds-pagination');
```

Comment:
not sure if it was intentional or not
```suggestion
      <Hds::Pagination @totalItems={{100}} @itemsPerPage={{10}} @currentPage={{1}} id="test-pagination" as |P|>
        <P.Nav />
      </Hds::Pagination>
    `);
    assert.dom('#test-pagination').hasClass('hds-pagination');
```